### PR TITLE
render QR Codes to terminal!

### PR DESCRIPTION
**Pull request by ewdurbin**
- ensure PyCrypto is available
- do some "pythonic" cleanup
- reduce verbosity of `__repr__` methods on custom classes
- QR Code rendering and demo!

### CODE REVIEW

File: decrypt_otpauth.py [hunk 1]
```diff
@@ -116,38 +124,6 @@ def decode_object(self, index):
         self.unpacked_uids[index] = obj
         return obj
 
-# @click.command()
-# @click.option('--encrypted-otpauth-account',
-#               help="path to your encrypted OTP Auth account (.otpauth)",
-#               required=True,
-#               type=click.File('rb'))
-# def main(encrypted_otpauth_account):
```
Comment (CooperRS) [inline decrypt_otpauth.py:124]: I'd rather keep this block because it documents how to open `.otpauth` files.

File: decrypt_otpauth.py [hunk 2]
```diff
@@ -167,20 +148,40 @@ def main(encrypted_otpauth_backup):
     data = data[:-data[-1]]
 
     # Decode wrapping archive
-    archive = DangerousUnarchive(data).top_object()
```
Comment (CooperRS) [inline decrypt_otpauth.py:170]: If we remove the use of `DangerousUnarchive` here: Do we still need the definition of this class?

**Review by CooperRS — CHANGES_REQUESTED** [review]
Thank you very much for the improvements! 🙂

Only thing I really would like to change is to keep the method for decrypting `.otpauth` files. It would be even better if we could have both methods at the same time. Unfortunately, my knowledge of Python is a bit behind. That's why I did not do that so far 🙈.

File: decrypt_otpauth.py (hunk 2 above)
Comment (ewdurbin) [reply decrypt_otpauth.py:170]: unfortunately `DangerousUnarchive` is still needed for the next "layer" of archives. https://github.com/ewdurbin/decrypt-otpauth-files/blob/772221e84ca0c040502473f5a1ce2b840ac98df6/decrypt_otpauth.py#L163

File: decrypt_otpauth.py (hunk 1 above)
Comment (ewdurbin) [reply decrypt_otpauth.py:124]: Fixed in https://github.com/CooperRS/decrypt-otpauth-files/pull/1/commits/42bb25916aa19dcf8abe3700e5636851db47e3f9

**Comment by ewdurbin** [comment]
All set for another review @CooperRS ✨

File: decrypt_otpauth.py (hunk 2 above)
Comment (CooperRS) [reply decrypt_otpauth.py:170]: Ah, I'm sorry, I missed that.

**Review by CooperRS — APPROVED** [review]
Thank you so much for your help! 🙂

**Pull request merged**